.eslintrc.js

```diff
@@ -1,121 +1,20 @@
-    'plugin:unicorn/recommended',
-    'plugin:node/recommended',
-    'plugin:fp/recommended',
-    'plugin:ava/recommended',
-    'plugin:you-dont-need-lodash-underscore/all',
-  ],
-  reportUnusedDisableDirectives: true,
+  plugins: ['fp'],
+  extends: ['@netlify/eslint-config-node', 'plugin:node/recommended', 'plugin:fp/recommended'],
   rules: {
-    'no-unused-vars': [2, {}],
-    'no-empty': [2, { allowEmptyCatch: true }],
-    'import/order': [
-      2,
-      {
-        'newlines-between': 'always',
-        alphabetize: {
-          order: 'asc',
-          caseInsensitive: true,
-        },
-      },
-    ],
-    'no-undef': [2, { typeof: true }],
-
-    'max-lines': [
-      2,
-      {
-        max: 150,
-        skipBlankLines: true,
-        skipComments: true,
-      },
-    ],
-    'max-lines-per-function': [
-      2,
-      {
-        max: 100,
-        skipBlankLines: true,
-        skipComments: true,
-        IIFEs: true,
-      },
-    ],
-    'max-statements': [2, 15],
-    'max-statements-per-line': [2, { max: 2 }],
-    'import/max-dependencies': [2, { max: 20 }],
-    complexity: [2, 5],
-    'max-depth': [2, 2],
-    'max-nested-callbacks': [2, 2],
-    'require-await': 2,
-
     // Those ESLint rules are not enabled by Prettier, ESLint recommended rules
     // nor standard JavaScript. However, they are still useful
-    'array-callback-return': [2, { allowImplicit: true, checkForEach: true }],
-    'block-scoped-var': 2,
     'class-methods-use-this': 2,
-    'consistent-this': 2,
-    'default-case': 2,
-    'default-case-last': 2,
-    'default-param-last': 2,
-    'func-name-matching': [2, { considerPropertyDescriptor: true }],
-    'func-names': [2, 'as-needed'],
-    'func-style': 2,
-    'line-comment-position': 2,
-    'max-classes-per-file': 2,
-    'max-params': [2, { max: 4 }],
-    'multiline-comment-style': [2, 'separate-lines'],
-    'no-await-in-loop': 2,
-    'no-bitwise': 2,
-    'no-constructor-return': 2,
-    'no-duplicate-imports': 2,
-    'no-else-return': [2, { allowElseIf: false }],
-    'no-extra-label': 2,
-    'no-implicit-coercion': 2,
-    'no-implicit-globals': [2, { lexicalBindings: true }],
-    'no-inline-comments': 2,
-    'no-invalid-this': 2,
-    'no-label-var': 2,
-    'no-lonely-if': 2,
-    'no-loop-func': 2,
-    'no-loss-of-precision': 2,
-    'no-magic-numbers': [
-      2,
-      {
-        ignore: [
-          // Common small numbers
-          -2,
-          -1,
-          0,
-          1,
-          2,
-          3,
-          // HTTP statuses
-          200,
-          201,
-          204,
-          300,
-          301,
-          400,
-          401,
-          403,
-          404,
-          410,
-          422,
-          500,
-        ],
-        enforceConst: true,
-        detectObjects: true,
-      },
-    ],
-    'no-multi-assign': 2,
-    'no-negated-condition': 2,
-    'no-nested-ternary': 2,
+    complexity: [2, 5],
+    'max-depth': [2, 2],
```

Comment:
The rules and plugins left are not shared by Netlify CLI yet, which is why they cannot be put in `@netlify/eslint-plugin-node` yet.